app/mailers/late_reservation_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class LateReservationMailer < ApplicationMailer
+	default from: 'notifications@example.com'
+
+	def forgot_to_buy_gift_email(user)
```

Comment:
two spaces. to avoid this in the future you can set sublime to handle tab/space translation: https://www.sublimetext.com/docs/2/indentation.html